src/renderer/vt/tracing.cpp

```diff
@@ -148,7 +148,8 @@ void RenderTracing::TraceStartPaint(const bool quickReturn,
                                     const til::bitmap invalidMap,
                                     const til::rectangle lastViewport,
                                     const til::point scrollDelt,
-                                    const bool cursorMoved) const
+                                    const bool cursorMoved,
+                                    std::optional<short> wrappedRow) const

```

Comment:
take this bb by const ref, even if only to save the copy in the case where tracing is turned off